docs/manual/java/guide/cluster/PersistentEntity.md

```diff
@@ -20,19 +20,21 @@ When an entity is started it replays the stored events to restore the current st
 
 ## Choosing a database
 
-Lagom supports the following databases:
+Lagom is compatible with the following databases:
 
-* [Cassandra](https://cassandra.apache.org/)
-* [PostgreSQL](https://www.postgresql.org/)
-* [MySQL](https://www.mysql.com/)
-* [Oracle](https://www.oracle.com/database/index.html)
-* [H2](https://www.h2database.com/)
+- [Cassandra](https://cassandra.apache.org/)
+- [PostgreSQL](https://www.postgresql.org/)
+- [MySQL](https://www.mysql.com/)
+- [Oracle](https://www.oracle.com/database/index.html)
+- [H2](https://www.h2database.com/)
+- [Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/)
+- [Couchbase](https://www.couchbase.com/)
 
-We recommend using Cassandra. Cassandra is a very scalable distributed database, and it is also flexible enough to support typical use cases of reactive services. In contrast to most relational databases, it natively supports sharding and replication, and is emerging as an industry standard open source NoSQL database.
+For instructions on configuring your project to use Cassandra, see [[Using Cassandra for Persistent Entities|PersistentEntityCassandra]]. If instead you want to use one of the relational databases listed above, see [[Using a Relational Database for Persistent Entities|PersistentEntityRDBMS]] on how to configure your project. If you wish to use Couchbase, proceed to the [Lagom section of the plugin site](https://doc.akka.io/docs/akka-persistence-couchbase/current/lagom-persistent-entity.html) for all the details.
```

Comment:
```suggestion
For instructions on configuring your project to use Cassandra, see [[Using Cassandra for Persistent Entities|PersistentEntityCassandra]]. If instead you want to use one of the relational databases listed above, see [[Using a Relational Database for Persistent Entities|PersistentEntityRDBMS]] on how to configure your project. If you wish to use Couchbase, proceed to the [Lagom section of the Akka Persistence Couchbase site](https://doc.akka.io/docs/akka-persistence-couchbase/current/lagom-persistent-entity.html) for all the details.
```

Reply:
The change buried in this blob of text is "plugin" --> "Akka Persistence Couchbase". Since there is no other immediate context for this, maybe people won't understand what "the plugin" refers to.